charts/redis-cluster/templates/redis-cluster.yaml

```diff
@@ -31,7 +31,9 @@ spec:
 {{- if eq .Values.pdb.enabled true }}
     pdb:
       enabled: {{ .Values.pdb.enabled }}
+      {{- if .Values.pdb.maxUnavailable }}
       maxUnavailable: {{ .Values.pdb.maxUnavailable }}
+      {{- if .Values.pdb.minAvailable }}
       minAvailable: {{ .Values.pdb.minAvailable }}
```

Comment:
You are missing the end statement here https://github.com/helm/charts/blob/master/stable/kubernetes-dashboard/templates/pdb.yaml#L16

Reply:
Fixed